quickstarts/aws/aws-translate/config.yml

```diff
@@ -0,0 +1,32 @@
+slug: aws-translate
+description: |-
+  ## Monitor your AWS Translate Metrics
+
+  With New Relic's Amazon Translate integration, you now can see all your Transate data on New Relic Instant Observability Platform.
+
+  ### Get started!
+
+   Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-translate/) to learn more about New
+   Relic monitoring and integration with Amazon Translate.
+
+summary: Use New Relic - Amazon Translate Integration to monitor your Translate utilizing Apps and Services with New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon Translate
+documentation:
+  - name: Amazon Translate installation docs
+    description: |
+      Monitor Amazon Translate by connecting AWS to New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-translate/
+keywords:
+  - aws
+  - amazon web services
+  - translate
+installPlans:
+  - aws-translate
+dataSourceIds:
```

Comment:
Please create this data source as it does not currently exist or update this to refer to the proper data source. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources

Reply:
added